frontend/src/features/apartment/ApartmentSalesPage.tsx

```diff
@@ -0,0 +1,296 @@
+                                </div>
+                                <div className="fieldset--max-prices__value">
+                                    <legend>Enimmäishinta per m² (€)</legend>
+                                    <span>{formatMoney(maxPrices.maxPricePerSquare)}</span>
+                                </div>
+                                <div className="fieldset--max-prices__value">
+                                    <legend>Velaton enimmäishinta (€)</legend>
+                                    <span>{formatMoney(maxPrices.debtFreeMaxPrice)}</span>
+                                </div>
+                            </div>
+                            <div className="row row--buttons">
+                                <p>
+                                    Enimmäishinnat <span>{formatDate(formData.dateOfCount)}</span> tehdystä
+                                    enimmäishintalaskelmasta (peruste:
+                                    <span> {formatIndex(data ? data.index : maxPriceQuery?.index)}</span>).
+                                </p>
+                                <Button
+                                    theme="black"
+                                    variant="secondary"
+                                    onClick={handleCalculateButton}
+                                >
+                                    Tee uusi enimmäishintalaskelma
+                                </Button>
+                            </div>
+                        </div>
+                    ) : (
+                        <div className="row row--prompt">
+                            <p>
+                                Asunnosta ei ole vahvistettua enimmäishintalaskelmaa, tai se ei ole enää voimassa. Syötä{" "}
+                                <span>kauppakirjan päivämäärä</span> sekä <span>yhtiön lainaosuus</span>, ja tee sitten
+                                uusi enimmäishintalaskelma saadaksesi asunnon enimmäishinnat kauppaa varten.
+                            </p>
+                            <Button
+                                theme="black"
+                                onClick={handleCalculateButton}
+                            >
+                                Tee enimmäishintalaskelma
+                            </Button>
+                        </div>
+                    )}
+                </Fieldset>
+                <Fieldset heading="Omistajuudet">
+                    <OwnershipsList
+                        formOwnershipsList={formOwnershipsList}
+                        setFormOwnershipsList={setFormOwnershipsList}
+                        apartment={data}
+                    />
+                </Fieldset>
+            </div>
+            <div className="row row--buttons">
+                <NavigateBackButton />
+                <SaveButton
+                    onClick={() => {
+                        return;
+                    }}
+                    isLoading={false}
```

Comment:
When we pass `isLoading={isLoading}` we can display the loading indicator on the `SaveButton`'s icon. If its not required the parameter should simply be left out (The component needs to be changed so it has a default value)